### PR TITLE
Add lib API to add cube & move object

### PR DESCRIPTION
**Pull request by taiki-e**

### CODE REVIEW

File: src/app.rs [hunk 1]
```diff
@@ -491,14 +492,46 @@ impl AppState {
                 self.point_cloud_renderer
                     .insert(points.id, &points.points, &points.colors);
             } else {
-                warn!(
-                    "points={},colors={}",
+                error!(
+                    "point cloud length mismatch points={},colors={}",
```
Comment (OTL) [inline src/app.rs:496]:
```suggestion
                    "point cloud length mismatch points={}, colors={}",
```

のがよさそう

**Review by OTL — APPROVED** [review]

File: src/app.rs [hunk 2]
```diff
@@ -512,22 +545,36 @@ impl AppState {
...
-            for i in 0..3 {
-                cur_ro.position[i] = o.translation.vector[i];
-            }
-            cur_ro.quaternion[0] = o.rotation.quaternion().w;
-            cur_ro.quaternion[1] = o.rotation.quaternion().i;
-            cur_ro.quaternion[2] = o.rotation.quaternion().j;
-            cur_ro.quaternion[3] = o.rotation.quaternion().k;
         }
+        let mut cur_ro = handle.current_robot_origin.lock();
+        let o = self.app.robot.origin();
```
Comment (OTL) [inline src/app.rs:570]: `o` seems too short to describe the origin. I prefer to use `origin`.

File: src/app.rs [hunk 3]
```diff
@@ -512,22 +545,36 @@ impl AppState {
...
-            let o = self.app.robot.origin();
-            for i in 0..3 {
-                cur_ro.position[i] = o.translation.vector[i];
-            }
-            cur_ro.quaternion[0] = o.rotation.quaternion().w;
-            cur_ro.quaternion[1] = o.rotation.quaternion().i;
-            cur_ro.quaternion[2] = o.rotation.quaternion().j;
-            cur_ro.quaternion[3] = o.rotation.quaternion().k;
         }
+        let mut cur_ro = handle.current_robot_origin.lock();
```
Comment (OTL) [inline src/app.rs:569]: I think `cur_origin` is better than `cur_ro`.

**Comment by taiki-e** [comment]
review addressed.

**Review by OTL — APPROVED** [review]

**Pull request merged**